tests/test_app.py

```diff
@@ -2,7 +2,10 @@
 import logging
 import time
 from unittest import TestCase
-import urllib
+try:
+    import urllib.parse as urllib  # For Python 3 compat
+except ImportError:
+    import urllib
 import uuid
 
```

Comment:
For the sake of readability I would group the imports such that first you have all `import...`, than `from ...`, and last the try-catch import.

Reply:
Fixed in acd9ca8!